Makefile

```
@@ -89,7 +89,7 @@ gen-grpc-resources:
 # Generate plugins YAML index files for both all plugins and end-user ones.
 gen-plugins-index: build-plugins
 	go run ./hack/gen-plugin-index.go -output-path ./plugins-dev-index.yaml
-	go run ./hack/gen-plugin-index.go -output-path ./plugins-index.yaml -plugin-name-filter 'kubectl|helm|kubernetes|prometheus|x|doctor|keptn'
+	go run ./hack/gen-plugin-index.go -output-path ./plugins-index.yaml -plugin-name-filter 'kubectl|helm|kubernetes|prometheus|exec|doctor|keptn|github-events'
```

```suggestion
	go run ./hack/gen-plugin-index.go -output-path ./plugins-index.yaml -plugin-name-filter 'kubectl|helm|kubernetes|prometheus|exec|doctor|keptn'
```

The `github-events` will be added in the next PR.